advertising/email-00-registration.md

```diff
@@ -0,0 +1,30 @@
+Title: Our Networks: registration open + vend at the Yami-ichi ⚑
+
+⚐ REGISTRATION
+We are excited to announce that registration is now open for State of Our Networks 2018.
+
+There are three sliding scale admission levels: Organization Supporter, Supporter, and Reduced Cost. The true cost for us to host this event is $57 per person. Your support at or above that price allows us to offer reduced cost admissions and use sponsorship money elsewhere. If it is difficult to attend even at a reduced cost, contact us at orga@ournetworks.ca. We are able to support a limited number of scholarships and are also looking for volunteers interested in helping at the event in return for complimentary admission. 
+
+https://ournetworks.ca/conference/#registration
+
+⚐ PROGRAM HIGHLIGHTS
+Our call for submissions closed with an overwhelming number of submissions. We’re just finalizing the schedule, but you can expect:
+
+▷ Talks about distributed data stewardship, migrating to peer-to-peer implementations, and the tactics used by existing protocols with a view to changing the internet
+▷ Discussion sessions on transitioning to decentralized projects and imagining that alternatives we want
```

Comment:
>Discussion sessions on transitioning to decentralized projects and imagining **_that_** alternatives we want 

_the_ alternatives?
_what_ alternatives?

Reply:
👍  I think it was "what"